filament/src/materials/dofUtils.fs

```diff
@@ -56,20 +53,14 @@ vec4 blurTexture(sampler2D colorBuffer, highp vec2 uv, highp vec2 direction, flo
     vec4 finalColor = textureLod(colorBuffer, uv, 0.0);
 
     highp vec2 unit = materialParams.resolution.zw;
-    direction *= unit * BLUR_SCALE;
-
-    float noise = 0.0;
-    if (BLUR_SCALE != 1.0) {
-        // we span 2 samples because the first sample is always fixed (no noise added)
-        noise = (random(gl_FragCoord.xy) - 0.5) * 2.0;
```

Comment:
Did you remove the random() function? Since you don't use it anymore

Reply:
good catch, thanks!